src/chocolatey/infrastructure.app/commands/ChocolateyInstallCommand.cs

```diff
@@ -233,7 +234,7 @@ public virtual void Validate(ChocolateyConfiguration configuration)
             if (!string.IsNullOrWhiteSpace(configuration.Input))
             {
                 var unparsedOptionsAndPackages = configuration.Input.Split(new[] { " " }, StringSplitOptions.RemoveEmptyEntries);
-                if (!configuration.Information.IsLicensedVersion)
+                if (!(configuration.Information.IsLicensedVersion && configuration.Information.IsLicensedAssemblyLoaded))

```

Comment:
I am not so sure about this change. Yes the Directory argument won't be available if they don't have the extension installed, but it is however available for a licensed customer, throwing an exception if the license installed doesn't sound appropriate.

At best I would say it should only output a warning in that case that they need to install a supported version of Chocolatey Licensed Extension.

@gep13 thoughts?

Reply:
As per our discussion on Zoom, I've backed out this change.